pyproject.toml

```diff
@@ -6,14 +6,14 @@ description = "The gateway for TACo application operations on the Threshold Netw
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4"
-nucypher = {git = "https://github.com/nucypher/nucypher.git", branch = "signing"}
+nucypher = {git = "https://github.com/kprasch/nucypher.git", branch = "erc4337+eip712"}
```

Comment:
This is temporal, right? I mean, using kprach repo.

Reply:
Correct - it was really a placeholder until https://github.com/nucypher/nucypher/pull/3598 was merged. Updated now to point to the `epic-signing` branch since the PR was merged.